src/services/namespaces/index.js

```diff
@@ -13,9 +13,4 @@ export const HYPERVIEW = 'https://hyperview.org/hyperview';
 export const HYPERVIEW_ALERT = 'https://hyperview.org/hyperview-alert';
 
 // Custom namespaces
```

Comment:
Perhaps this comment is now outdated? Since we've moved out all "custom" 3rd party behavior out of this lib

Reply:
the `SHARE` namespace is still "custom" (url being on instawork.com instead of hyperview.org)